### ISSUE TITLE
Please don't generally require OpenGL when only tests use it

### ISSUE THREAD

**Issue by yurivict** [sadko4u/lsp-plugins#14]
Only tests use it:
```
$ grep -r GL/ work | grep inclu
work/lsp-plugins-e29c646/src/test/mtest/core/geometry3d.cpp:#include <GL/gl.h>
work/lsp-plugins-e29c646/src/test/mtest/core/geometry3d.cpp:#include <GL/glx.h>
work/lsp-plugins-e29c646/src/test/mtest/core/geometry3d.cpp:#include <GL/glu.h>
```

But the root ```Makefile``` requires it unconditionally:
```
$ grep -r OPENGL work/
work/lsp-plugins-e29c646/Makefile:export OPENGL_HEADERS   = $(shell pkg-config --cflags gl glu)
work/lsp-plugins-e29c646/Makefile:export OPENGL_LIBS      = $(shell pkg-config --libs gl glu)
work/lsp-plugins-e29c646/src/container/Makefile:  LIBS                   += $(OPENGL_LIBS)
work/lsp-plugins-e29c646/src/core/Makefile:	@$(CC) -o $(@) -c $(FILE) -fPIC $(CPPFLAGS) $(CFLAGS) $(INCLUDE) $(OPENGL_HEADERS) 
work/lsp-plugins-e29c646/src/test/Makefile:	@$(CC) -o $(@) -c $(FILE) -fPIC $(CPPFLAGS) $(CFLAGS) $(INCLUDE) $(OPENGL_HEADERS) 
```

Please only require it in the ```test``` folder.

**Comment by sadko4u** [comment]
Currently you're right, only tests do use OpenGL but in the future I think it will be a required option for some plugins or at least for 3D rendering engine.
Anyway I'll remove this dependency for **devel** branch.

**Comment by sadko4u** [comment]
Created **github-issue-14** branch.
This commit should fix the problem: da2f9b1ed06ae4255e78dd3ecaaec44e4b4365e2

**Comment by sadko4u** [comment]
Merged changes into **devel** branch: fe2a6fbea00d27bbc14fe73502b6b8538ad08ddc
Closing issue
